packages/cdktf-cli/src/bin/cmds/deploy.ts

```diff
@@ -84,6 +84,12 @@ class Command extends BaseCommand {
         required: false,
         desc: "Disables terminal formatting sequences in the output.",
       })
+      .option("migrate-state", {
+        type: "boolean",
+        default: false,
+        required: false,
+        desc: "Reconfigure a backend, and attempt to migrate any existing state for all selected stacks.",
```

Comment:
Maybe reword this to
```suggestion
        desc: "Pass this flag after switching state backends to approve a state migration for all targeted stacks",
```